pkg/pool-weighted/contracts/managed/CircuitBreakerStorageLib.sol

```diff
@@ -168,72 +170,67 @@ library CircuitBreakerStorageLib {
      * given by: B1 = k * B0. Since the BPTPrice0 = totalSupply*weight/B0, and BPTPrice1 = totalSupply*weight/B1,
      * we can combine these equations to compute the BPT price ratio BPTPrice1/BPTPrice0 = 1/k; BPT1 = BPT0/k.
      * So we see that the "conversion factor" between the spot price ratio and BPT Price ratio can be written
-     * as above BPT1 = BPT0 * (1/k), or more simply: (reference BPT price) * (priceRatio)**(1 - weight).
+     * as above BPT1 = BPT0 * (1/k), or more simply: (BPT price) * (priceRatio)**(1 - weight).
      *
-     * If the value of the weight complement has not changed, we can use the reference conversion ratios stored
+     * If the value of the weight complement has not changed, we can use the cached weight adjustments stored
```

Comment:
```suggestion
     * If the value of the weight complement has not changed, we can use the cached adjusted bounds stored
```

"weight adjustments" to me implies a delta rather than the final value.